cli/enable/test_data/test_script/test.lua

```diff
@@ -0,0 +1,11 @@
+local inst_name = os.getenv('TARANTOOL_INSTANCE_NAME')
```

Comment:
Do we need real script to check symlink creation?

Reply:
changed